.NET/Samples/SimpleConsole/Program.cs

```diff
@@ -38,9 +39,10 @@ static void Main(string[] args)
                     if (input.Length > 0)
                     {
                         // Retrieve all the parsers and call 'Parse' to recognize all the values from the user input
-                        var results = GetModels()
+                        /*var results = GetModels()
```

Comment:
Please follow the same usage pattern. And do not push commented out code to the repo.